### PR TITLE
[Enhancement] Add windows-cuda-build ci

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

Add windows-cuda-trt build check

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -207,3 +207,143 @@ jobs:
           env_vars: OS,PYTHON,CPLUS
           name: codecov-umbrella
           fail_ci_if_error: false
+
+  build_cuda102-windows:
```
Comment (RunningLeon) [inline .github/workflows/build.yml:211]: the newly added ci failed.

[the remark this answers is not in the document]
Comment (irexyc) [reply .github/workflows/build.yml:211]: I think it's better not to combine since it takes  long time to build and two tasks can run in parallel.

Comment (irexyc) [reply .github/workflows/build.yml:211]: will be fixed by https://github.com/open-mmlab/mmdeploy/pull/1626

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -207,3 +207,143 @@ jobs:
...
+          Invoke-WebRequest -Uri https://github.com/irexyc/mmdeploy-ci-resource/releases/download/pplcv/pplcv-0.6.2.windows10.x86_64.cuda-10.2.zip -OutFile pplcv.zip
+          Expand-Archive pplcv.zip pplcv
+          echo "PPLCV_DIR=$pwd\pplcv\lib\cmake\ppl" >> $env:GITHUB_ENV
+      - name: Build
+        run: |
+          mkdir build
+          cd build
+          cmake ..  -A x64 -T v142 `
+            -DMMDEPLOY_SHARED_LIBS=OFF `
+            -DMMDEPLOY_BUILD_SDK_CXX_API=ON `
```
Comment (lvhan028) [inline .github/workflows/build.yml:255]: `MMDEPLOY_BUILD_SDK_CXX_API` option has been removed.

**Pull request closed without merging**